PacketDelivery-ReferenceExample/Data-Service-Provider/values/values-orion.yml

```diff
@@ -0,0 +1,40 @@
+broker:
+  envPrefix: ORIONLD_
+  mutexPolicy: none
+  maxConnections: 20000
+  logging:
+    level: FATAL
+  reqPoolSize: 8
+  # use this config for EntityUpdateWithTypeSubscriptionSimulation
+  notificationMode: persistent
+  # use this config for EntityUpdateWithSubscriptionSimulation
+  # second number should be around the expected clients at one instance, first is a multiple of second
+  # notificationMode: threadpool:10000:1000
+  db:
+    hosts:
```

Comment:
Didn't  see that before, but do we really need 4 replicas? 2 is probably sufficient.

Reply:
Will reduce to standalone